modules/doc/content/newsletter/2020_04.md

```diff
@@ -1 +1,20 @@
 # MOOSE Newsletter (April 2020)
+
+## SoftwareX Paper Published
+
+The new MOOSE paper was published in SoftwareX: [MOOSE: Enabling massively parallel multiphysics simulation](https://doi.org/10.1016/j.softx.2020.100430).
+
+## Update `validParams`
+
+The framework and all modules have been updated to use a static function for `validParams`. Please
+update your applications to use the new static function and see
+[#14135](https://github.com/idaholab/moose/issues/14135) see the changes in MOOSE.
```

Comment:
The issue referenced here doesn't actually give any guidance about what an app developer is supposed to do. I assume we are supposed to only declare and implement the static `validParams()` method, get rid of the templated version, and also get rid of the forward declaration of the current class (which is generally only needed for the templated `validParams()`). We should just provide those details right here.

Reply:
I was just lazy, I added a detailed explanation.

Reply:
Thanks, very helpful now.